update_donations.mjs

```diff
@@ -44,7 +44,7 @@ const donationData = [
   },
   {
     name_: "patreon",
-    patrons: patreonData.data[0].attributes.patron_count,
+    patrons: patreonData.data[0].attributes.paid_member_count,
```

Comment:
There's no such field: https://docs.patreon.com/#campaign

Reply:
Its definitely returning this field right next to patron_count, but apparently its not documented.